test/lib/.eslintrc

```diff
@@ -0,0 +1,6 @@
+extends:
```

Comment:
do you really need to put these at each directory level or is the one in `test/` enough?